doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -444,16 +444,21 @@ Cellular samples (renamed from nRF9160 samples)
 
 * :ref:`lwm2m_client` sample:
 
-  * Added an overlay for using DTLS Connection Identifier.
-    This significantly reduces the DTLS handshake overhead when doing the LwM2M Update operation.
+  * Added:
+
+    * Including a overlay for using DTLS Connection Identifier.
+      This significantly reduces the DTLS handshake overhead when doing the LwM2M Update operation.
 
-    * An overlay for using DTLS Connection Identifier.
+      * An overlay for using DTLS Connection Identifier.
       This significantly reduces the DTLS handshake overhead when doing the LwM2M Update operation.
-    * Support for Hosting MCUmgr client for external MCU.
+      * Support for Hosting MCUmgr client for external MCU.
       Including a new overlay file for enabling this and devicetree overlay files for UART2 and MCUboot recovery mode.
+    * Including a overlay for enabling proprietary Power Saving Mode (PSM).
+      This will fix a case when battery operated device join network which not support PSM.
```

Comment:
```suggestion
    * Support for Hosting MCUmgr client for external MCU.
      Including a new overlay file for enabling this and devicetree overlay files for UART2 and MCUboot recovery mode.
    * An overlay for enabling proprietary Power Saving Mode (PSM).
      This will fix a case where a battery-operated device joins a network that does not support PSM.
```